tests/purs/failing/OverlappingUnnamedInstances.out

```diff
@@ -0,0 +1,22 @@
+Error found:
+in module [33mMain[0m
+at tests/purs/failing/OverlappingUnnamedInstances.purs:10:1 - 11:13 (line 10, column 1 - line 11, column 13)
+
+  Overlapping type class instances found for
+  [33m               [0m
+  [33m  Main.Test Int[0m
+  [33m               [0m
+  The following instances were found:
+
+    Main.$test1
```

Comment:
This probably shouldn't happen in this PR, but now that instance names are optional we should probably include source spans in this list so that people can find the problematic instances a little more easily.